packages/rest/test/unit/router/reject.provider.unit.ts

```diff
@@ -28,6 +28,18 @@ describe('reject', () => {
     expect(result).to.have.property('statusCode', 500);
   });
 
+  it('converts error code ENTITY_NOT_FOUND to status code 404', async () => {
+    const reject = new RejectProvider(noopLogger).value();
+
+    const notFoundError: Error & {code?: string} = new Error('not found');
```

Comment:
I thought the `Error` type def would have `code` key in there. Maybe it's not optional and that's why we are defining the type of `notFoundError` like this?

Reply:
`code` isn't part of a standard Node `Error` object but something we add to it ourselves IIUC.

Reply:
As @virkt25 wrote, JavaScript `Error` class does not have any `code` property by default.

Reply:
Yes, no `code` in JavaScript `Error`.